ui-core/js/services/engine.ts

```diff
@@ -228,13 +228,17 @@ function initializeSimpleWithAuthorization(
     let streamId = null;
     let initializationResponse = null;
 
+    const queryParameters = Utils.parseQueryString(window.location.search.substring(1).toLowerCase());
```

Comment:
@griff-10 I think the parameter parsing exists at a layer above this already, so you probably want to pass in `environment` to `initializeSimpleWithAuthorization`. And anywhere else its needed

Reply:
Done